a3p-integration/package.json

```diff
@@ -12,5 +12,6 @@
         "@agoric/synthetic-chain": "^0.0.4-2",
         "tsx": "^4.7.0"
     },
+    "packageManager": "yarn@4.0.2",
```

Comment:
I'm concerned this will force anyone / anything running this to use `corepack`. Is it even enabled in CI?

Reply:
> I'm concerned this will force anyone / anything running this to use corepack.

Valid concern. I adopted it to make it possible to `yarn link` to a local checkout of `agoric-3-proposals`. I'm open to cutting it. But I would argue that we should adopt corepack throughout our development. 

> Is it even enabled in CI?

prolly not. easy to solve for this job

Reply:
> I adopted it to make it possible to `yarn link` to a local checkout of `agoric-3-proposals`.

I see. Technically a `link` is irrespective of the package manager, as it's just a symlink in `node_modules`.

Would temporarily overriding the "version" to a local path not work? Arguably this package is not part of the workspace, so my concerns are not high as long as we enable corepack in CI for this test.


> I would argue that we should adopt corepack throughout our development.

Yes we should get off the yarn1 train, but that's another topic.

Reply:
> Technically a link is irrespective of the package manager, as it's just a symlink in node_modules.

That's not how Yarn 2+ works. It uses `resolutions` in package.json. agoric-3-proposals is using 2+ so it can't produce what a3p-integration with Yarn 1 is expecting. Of course I could manually create the symlink but I thought it would be good progress to bump Yarn here.

> so my concerns are not high as long as we enable corepack in CI for this test

I'm hearing that as the request and agreement to maintain the Yarn version bump.

Reply:
for ref:
 - #451